docs/pvacseq/errors.rst

```diff
@@ -0,0 +1,169 @@
+run`` command to match one of them.
+
+**normal_sample_name not a sample ID in the #CHROM header of VCF**
+
+Your ``pvacseq run`` command included the ``--normal-sample-name`` parameter.
+However, the argument chosen did not match any of the sample identifiers in
+the ``#CHROM`` header of the input VCF.
+
+Take, for example, the following ``#CHROM`` VCF header:
+
+.. code-block:: none
+
+   #CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO	FORMAT	NORMAL	TUMOR
+
+All columns after ``FORMAT`` are sample identifiers that can be used as the
+``--normal-sample-name`` parameter when running pVACseq, depending on which
+sample is the normal sample in the VCF. Change the ``--normal-sample-name`` parameter of your ``pvacseq
+run`` command to match the appropriate sample identifier.
+
+**VCF doesn't contain any sample genotype information.**
+
+pVACseq uses the sample genotype to identified which variants were called.
+Therefore, while a VCF with a ``FORMAT`` and sample column(s) is valid, it cannot be used
+in pVACseq. You will need to manually edit your VCF and add a ``FORMAT`` and
+sample column with the ``GT`` genotype field. For more information on this
+formatting please see the `VCF specification <https://github.com/samtools/hts-specs>`_ for your specific VCF version.
+
+Input VCF Compression and Indexing
+__________________________________
+
+**Input VCF needs to be bgzipped when running with a proximal variants VCF.**
+
+When running pVACseq with the ``--proximal-variants-vcf`` argument, the main
+input VCF needs to be bgzipped and tabix indexed. See :ref:`the Input File
+Preparation section <bgzip_input_vcf>` of the documentation for instructions on how to do so.
+
+**Proximal variants VCF needs to be bgzipped.**
+
+The VCF provided via the ``--proximal-variants-vcf`` argument needs to be
+bgzipped and tabix indexed. See :ref:`the Input File
+Preparation section <bgzip_phased_vcf>` of the documentation for instructions on how to do so.
+
+**No .tbi file found for input VCF. Input VCF needs to be tabix indexed if processing with proximal variants.**
+
+When running pVACseq with the ``--proximal-variants-vcf`` argument, the main
+input VCF needs to be bgzipped and tabix indexed. See :ref:`the Input File
+Preparation section <bgzip_input_vcf>` of the documentation for instructions on how to do so.
+
+**No .tbi file found for proximal variants VCF. Proximal variants VCF needs to be tabix indexed.**
+
+The VCF provided via the ``--proximal-variants-vcf`` argument needs to be
+bgzipped and tabix indexed. See :ref:`the Input File
+Preparation section <bgzip_phased_vcf>` of the documentation for instructions on how to do so.
+
+Input VCF VEP Annotation
+________________________
+
+**Input VCF does not contain a CSQ header. Please annotate the VCF with VEP before running it.**
+
+pVACseq requires the input VCF to be annotated by VEP. The provided input VCF
+doesn't contain a ``CSQ`` ``INFO`` header which indicates that it has not been
```

Comment:
This sentence is a little confusing in that the `CSQ` header indicates that the VCF *has* been annotated, but the sentence is talking about how a lack of it indicates that is has not.  It's easy to read this as saying the opposite of what it does.

I'd at least add a comma:
```suggestion
doesn't contain a ``CSQ`` ``INFO`` header, which indicates that it has not been
```
Maybe more separation would help:
```suggestion
doesn't contain a ``CSQ`` ``INFO`` header. This indicates that it has not been
```